js/siteorigin-panels/view/builder.js

```diff
@@ -88,21 +88,21 @@ module.exports = Backbone.View.extend( {
 		} );
 		
 		// When the data changes in the model, store it in the field
-		this.model.on( 'change:data load_panels_data', this.storeModelData, this );
+		this.listenTo( this.model, 'change:data load_panels_data', this.storeModelData );
+		this.listenTo( this.model, 'change:data load_panels_data', this.toggleWelcomeDisplay );
 		
 		// Handle a content change
 		this.on( 'content_change', this.handleContentChange, this );
```

Comment:
These can also use `this.listenTo`.

Reply:
Can something use `listenTo` when listening to itself? The function is `object.listenTo(other, event, callback)` and the description starts with:

>Tell an object to listen to a particular event on an other object.

So shouldn't this still be using `on`?

Reply:
Ah, yeah, I think you're right. Using `this.listenTo` on `this` will either not work or just have the same effect as calling `this.on`.